brooklyn-docs/guide/start/running.md

```diff
@@ -7,33 +7,64 @@ menu_parent: index.md
 
 This guide will walk you through deploying an example 3-tier web application to a public cloud, and demonstrate the autoscaling capabilities of the Brooklyn platform.
 
-An overview of core [Brooklyn concepts](./concept-quickstart.html) is available for reference.
+An overview of core [Brooklyn concepts](./concept-quickstart.html){:target="_blank"} is available for reference.
 
-This tutorial assumes that you are using Linux or Mac OS X.
+Two methods of deployment are detailed in this tutorial, using virtualisation with Vagrant and a local install. Both assume that you are using Linux or Mac OS X.
 
 ## Install Apache Brooklyn
 
-Download the Apache Brooklyn binary distribution as described on [the download page]({{site.path.website}}/download/).
+{::options parse_block_html="true" /}
+
+<ul class="nav nav-tabs">
+    <li class="active impl-1-tab"><a data-target="#impl-1, .impl-1-tab" data-toggle="tab" href="#">Vagrant</a></li>
+    <li class="impl-2-tab"><a data-target="#impl-2, .impl-2-tab" data-toggle="tab" href="#">Local Install</a></li>
+</ul>
+
+<div class="tab-content">
+<div id="impl-1" class="tab-pane fade in active">
+
+[Vagrant](https://www.vagrantup.com/){:target="_blank"} is a software package which automates the process of setting up virtual environments such as [Oracle VirtualBox](https://www.virtualbox.org){:target="_blank"}. We recommend it as
+the easiest way of getting started with Apache Brooklyn.
+
+Firstly, download and install:
+
+ * [Vagrant](http://www.vagrantup.com/downloads){:target="_blank"}
+ * [Oracle VirtualBox](https://www.virtualbox.org/wiki/Downloads){:target="_blank"}
+ 
+Then download the default Apache Brooklyn vagrant configuration from [here](https://github.com/johnmccabe/vagrant-brooklyn-getting-started/archive/master.tar.gz){:target="_blank"}. This configuration contains everything you need to get started using Apache Brooklyn.
```

Comment:
can we move this to be a brooklyn project rather than in @johnmccabe 's repo?

not essential as part of this PR but i think it is important for apache that our primary getting-started path is pure apache -- or other well-known sources -- not some random guy, no matter how swell he is!